File: TipoTarefa.java
public enum TipoTarefa {
    ATIVIDADE("atividade"),
    TRABALHO("trabalho"),
    PROVA("prova");
    
    private final String valor;
    
    TipoTarefa(String valor) {
        this.valor = valor;
    }
    
    public String getValor() {
        return valor;
    }
}

File: tipoTarefa.py
from enum import Enum

class TipoTarefa(Enum):
    ATIVIDADE = "atividade"
    TRABALHO = "trabalho"
    PROVA = "prova"

    def __init__(self, valor):
        self._valor = valor

    @property
    def valor(self):
        return self._valor